packages/cli/src/commands/bisect/index.ts

```diff
@@ -370,6 +366,16 @@ export default async function main(client: Client): Promise<number> {
   return 0;
 }
 
+function hasScheme(url: string): Boolean {
+  return url.startsWith('http://') || url.startsWith('https://');
+}
+
+function validateUrl(url: string): string {
+  if (!hasScheme(url)) {
+    return `https://${url}`;
+  } else return url;
```

Comment:
```suggestion
  }
  return url;
```

No `else` needed here because of the early return

Reply:
Actually, you can do it with a single return

```ts
return hasScheme(url) ? url : `https://${url}`
```